connectors/bitbucket-server/src/test/resources/bitbucket/responses/success.json

```diff
@@ -4,10 +4,7 @@
       "id": "00000000-0000-0000-0000-000000000000",
       "creation_date": "1970-01-01T00:00:00Z",
       "header": {
-        "title": "Bitbucket Server PR - Feature/APF-651 add image and test auth test/Sudharsan Sathiamoorthy",
-        "subtitle": [
-          "Description - * APF-651-Added integration test for getImage and getAuthKey\r\n* APF-651-Added integration test for test auth for connectors"
-        ]
+        "title": "Bitbucket Server PR - Feature/APF-651 add image and test auth test/Sudharsan Sathiamoorthy"
```

Comment:
If you're going to change this, make it more like what I did with Github/Gitlab a few days ago:

```
"title": "Bitbucket Server Pull Request",
"subtitle": [
    "{projectKey}/{repositorySlug} #{pullRequestId}"
]
```

* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/github-pr/src/main/java/com/vmware/connectors/github/pr/GithubPrController.java#L231
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/github-pr/src/main/resources/cards/text.properties#L2
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/gitlab-pr/src/main/java/com/vmware/connectors/gitlab/pr/GitlabPrController.java#L252
* https://github.com/vmware/connectors-workspace-one/blob/master/connectors/gitlab-pr/src/main/resources/cards/text.properties#L2

Reply:
Added subtitle to the header in the format ```{projectKey}/{repositorySlug} #{pullRequestId}```.